README.md

```diff
@@ -444,7 +451,7 @@ Then you should be able to run the setup script:
 ```sh
 ./setup.sh
 ```
-With this other script you can choose the number of nodes that you want to set up:
+With this other script using tmux you can choose the number of nodes that you want to set up:
```

Comment:
Be sure to format the lines at 80 chars (In VS code the [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) plugin can be of a great help):

```
With this other script using tmux you can choose the number of nodes that you
want to set up:
```